tests/Feature/SubscriptionsTest.php

```diff
@@ -217,27 +205,14 @@ public function test_subscriptions_can_retrieve_their_payment_info()
 
         $this->assertSame('john@example.com', $subscription->paddleEmail());
         $this->assertSame('card', $subscription->paymentMethod());
-        $this->assertSame('visa', $subscription->cardBrand());
-        $this->assertSame('1234', $subscription->cardLastFour());
-        $this->assertSame('04/2022', $subscription->cardExpirationDate());
+        $this->assertSame($brand, $subscription->cardBrand());
+        $this->assertSame($last4, $subscription->cardLastFour());
+        $this->assertSame($expiry, $subscription->cardExpirationDate());
```

Comment:
I think I'd like to explicitly not make use of variables for this and duplicate the checked data.